specification/consumption/resource-manager/Microsoft.Consumption/stable/2023-03-01/consumption_pricesheet.json

```diff
@@ -0,0 +1,158 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-03-01",
+    "title": "ConsumptionManagementClient",
+    "description": "Consumption management client provides access to consumption resources for Azure Enterprise Subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingPeriods/{billingPeriodName}/providers/Microsoft.Consumption/pricesheets/download": {
+      "post": {
+        "tags": [
+          "PriceSheet"
+        ],
+        "operationId": "PriceSheet_DownloadByBillingAccountPeriod",
+        "description": "Generates the pricesheet for the provided billing period asynchronously based on the enrollment id",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/consumption/"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "EAPriceSheetForBillingPeriod": {
+            "$ref": "./examples/EAPriceSheetForBillingPeriod.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingPeriodNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request processing completed.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string"
+              },
+              "Retry-After": {
+                "type": "integer",
+                "format": "int64"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
Please consolidate the usage of `ErrorResponse`: https://github.com/Azure/azure-rest-api-specs/blob/572953114406caf6cc8bac3a9e745e8c43218993/specification/consumption/resource-manager/Microsoft.Consumption/stable/2023-03-01/consumption.json#L4967

Reply:
consumption.json is used by other API teams, hence I added a separate file consumption_pricesheet.json for changes related to my team's API. Hence not modifying the other file as part of this PR. Thanks!